gradle/scripts/lib/scala.gradle

```diff
@@ -58,7 +58,7 @@ configure(scala212 + scala213 + scala3) {
 
 configure(scala212) {
     dependencies {
-        implementation('org.scala-lang:scala-library:2.12.16')
+        implementation('org.scala-lang:scala-library:2.12.17')
```

Comment:
These changes are applied to `gradle-scripts` via https://github.com/line/gradle-scripts/pull/129